packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -194,15 +194,19 @@ export function ArticleEditor({id}: ArticleEditorProps) {
       setTagTitle(
         t('articleEditor.overview.pending', {
           date: new Date(articleData?.article?.pending?.publishAt ?? '').toDateString(),
-          time: new Date(articleData?.article?.pending?.publishAt ?? '').toLocaleTimeString()
+          time: new Date(articleData?.article?.pending?.publishAt ?? '')
```

Comment:
With the formatting options in i18next it should be possible to pass just one date object and do the formatting in the translation files.